README.md

```diff
@@ -76,9 +80,9 @@ After/during registering, we need to update FFmpeg in order to have a reference
 | IANA      | Matroska |               |               |               |                                        |
 | IANA      | FFV1     |               |               |               |                                        |
 | IANA      | FLAC     |               |               |               |                                        |
-| MP4       | FFV1     |               |               |               |                                        |
+| MP4       | FFV1     | 2017-02-14    |               |               | [Proposal](mp4ra_FFV1.md)              |
```

Comment:
I was thinking about date of first contact, not date of the draft. I suggest to remove the date part until we send the email to mp4ra.